File: c/p006.c
#include <stdio.h>
#include <math.h>

int main()
{
    int n = 100;
    long diff = pow(((n * (n+1)) / 2), 2) - ((2*pow(n, 3) + 3*pow(n, 2) + n)/ 6);
    printf("%ld", diff);
}

File: java/p006.java
public class p006{
    public static void main(String[] args) {
        System.out.println(p006.run());
    }

    public static int run() {
        int sumOfSquares = 0;
        int squareOfSum = 0;
        for (int i = 1; i <= 100; i++) {
            sumOfSquares += Math.pow(i, 2);
            squareOfSum += i;
        }
        squareOfSum = (int) Math.pow(squareOfSum, 2);
        return squareOfSum - sumOfSquares;
    }
}